graylog2-server/src/main/java/org/graylog/grn/GRNRegistry.java

```diff
@@ -33,7 +33,10 @@
  */
 @Singleton
 public class GRNRegistry {
-    public static final String GLOBAL_USER_GRN = "grn::::builtin-team:everyone"; // TODO: Find a better name for the "everyone" grantee GRN type
+    // TODO: Find a better name for the "everyone" grantee GRN type
+    public GRN GLOBAL_USER_GRN() {
```

Comment:
Why are you converting this into a method instead of keeping it as a constant? If we use a method, we should adhere to the method naming conventions.

Reply:
yeah, not sure what I did there :-D